components/ProjectCard.js

```diff
@@ -30,5 +34,4 @@ const ProjectItem = styled.li`
   list-style-type: none;
   width: 85%;
   border-radius: 5px;
```

Comment:
Same with border-radius, I was advised by the coaches to use `em` or `rem` there